anatomy/tracks/configlet/linting.md

```diff
@@ -1 +0,0 @@
-This file has moved to https://github.com/exercism/docs/blob/main/building/configlet/lint.md
```

Comment:
Please don't remove this file, as it is there to gracefully handle old links.

Reply:
Removal has been reverted